GVFS/GVFS.Common/Git/DiffTreeResult.cs

```diff
@@ -153,6 +157,7 @@ public static DiffTreeResult ParseFromLsTreeLine(string line, string repoRoot)
                 {
                     DiffTreeResult blobAdd = new DiffTreeResult();
                     blobAdd.TargetIsSymLink = line.StartsWith("120000");
+                    blobAdd.TargetMode = line.Substring(0, 6);

```

Comment:
Minor, do you think these two lines would be easier to follow if the `StartsWith` was removed?

```
blobAdd.TargetMode = line.Substring(0, 6);
blobAdd.TargetIsSymLink = blobAdd.TargetMode == "120000";
```